src/screens/MemberDetail/MemberDetail.tsx

```
@@ -253,11 +263,76 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
+              style={{ margin: 'auto', fontSize: '1rem' }}
+              className={`text-gray-100`}
+            >
+              {t('navigationOrganisations')}
+            </h3>
+          </button>
+        </a>
+
+        <a href="/orgevents">
+          <button
+            className={`${styles.featuresBox} ${styles.allRound} flex items-center rounded-lg text-gray-100 transition-colors`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)